lib/services/create_basket.rb

```diff
@@ -0,0 +1,10 @@
+require_relative "./../model/basket"
+
+module Store
+  class CreateBasket
+     def call
+        BASKET << Basket.new
+        BASKET[-1]
```

Comment:
it's not basket, but more item in basket. And what is the reason behind those method? Basket should be just `[]`, for what you need empty item?